### DISCUSSION TITLE
HLS video stream listener statistics

### DISCUSSION THREAD

**Discussion by gAlleb** [AzuraCast/AzuraCast#6805] [Q&A]
Hi! I was wondering is it possible to turn on `statistics` for custom HLS video streams? 

For custom icecast video streams it is pretty straightforward — https://github.com/AzuraCast/AzuraCast/issues/6774#issuecomment-1858927727. What about HLS? If it is not — maybe we should think about introducing such feature.

Thanks!

**Comment by BusterNeece** [comment]
We do have statistics for the HLS stream support that's built into AzuraCast. We don't currently support collecting stats on custom streams as we depend on deep integration with Nginx to pull the stats we use on our own HLS stream.

**Comment by Moonbase59** [comment]
That’s what I feared. Pity, would have been nice, considering bandwidth adjusting for video. (IF—and that’s a big if—one has the resources to generate multiple-resolution video on the server. Almost impossible without hardware acceleration.)
